Regression/WarpX-tests.ini

```diff
@@ -465,6 +490,7 @@ compileTest = 0
 doVis = 0
 compareParticles = 1
 particleTypes = electrons positrons
+outputFile = diags/d00040
```

Comment:
Could you explain why has this line been added to many tests?

Reply:
Yes, I encountered issues while running the test suite for these tests that are now using the new diagnostics (because their input files have been modified to do so). More precisely, the default output file name with the new diagnostics is `diags/d00040`. By adding `d.file_prefix=diags/plotfiles/plt` to the input files, the default output file name becomes `diags/plotfiles/plt00040`, which matches the default WarpX output file names convention. However, the test suite still fails to find the correct output file. The reason is that, for a test named `test_name`, the test suite seems to look for an output file named `test_name_plt00040`, unless `outputFile` is specified in the INI file. This behavior should be evident by looking at the various Python scripts used to run this suite. The reason why all other tests work well without specifying `outputFile` in the INI file is that the execution command to run the test suite contains always the **default runtime parameter** (not defined in the INI file) `amr.plot_file=test_name_plt` (example for the test named `test_name`), which overwrites the default WarpX output file name prefix `diags/plotfiles/plt`. An output file name prefix such as `test_name_plt` is specific for one single test, while input files are usually shared among multiple tests, hence I cannot add to the input files lines such as `d.file_prefix=test_name_plt` (which would fix the issue only if each input file was used by exactly one single test).

Reply:
I added the customized output file name prefix as a runtime parameter in the INI file (hence without changing the input files) just for the two new tests using the current correction. This way I do not need to change `outputFile` for the old tests not using the current correction but sharing the same input files. Relating to my message above, I think that the correct fix would be to at least have the same default runtime parameter assignment (as for `amr.plot_file` when we run the test suite at the moment) for the output file name prefix of each new diagnostic `d#.file_prefix`, so that it doesn't have to be done every time in the INI file for each new test. This should probably be discussed more and be addressed in a different PR.

Reply:
Ok thanks for the check, we could probably agree upon a given diagnostics name e.g. `test_diags` for the tests and add a `name` option to the new diags, so that we could replace `amr.plot_file=test_name_plt` with `test_diags.name=test_name_plt`.

Reply:
Should I do this in a separate PR and merge it before this one, so that it can be then used here?